scripts/release.ts

```diff
@@ -308,10 +312,20 @@ async function handler({
     process.exit(EXIT_CODES.SUCCESS);
   }
 
-  logger.log();
-  logger.log(`Pushing changes to Github...`);
-  await exec(gitCommand);
-  logger.log(`Changes pushed successfully`);
+  if (isExperimental) {
+    logger.log();
+    logger.log(`Experimental release: Skipping push to Github`);
+
+    // Reset the version back to the local version
+    await exec(
+      `yarn version --no-git-tag-version --new-version ${localVersion}`
+    );
```

Comment:
If we don't reset this back to the `localVersion` the change will still be in the package.json